htdocs/install/mysql/tables/llx_inventory-stock.sql

```diff
@@ -28,8 +28,9 @@ CREATE TABLE llx_inventory
   fk_user_modif integer,                    -- user making last change
   fk_user_valid integer,                    -- valideur de la fiche
   fk_warehouse integer DEFAULT NULL, 
-  fk_product integer DEFAULT NULL, 
-  status integer DEFAULT 0, 
+  fk_product integer DEFAULT NULL,
+  fk_categories_product varchar(255) DEFAULT NULL, -- product categories id separated by comma
```

Comment:
Because syntax fk_... is reserved for foreign key that point to a primary key, can you rename the field into just "categories_product".
PS: it is recommended to push PR on SQL structure change into a different PR so structure can be quickly validated and merged before the code is done.